### PR TITLE
Change height of svg of animals and exchange big dog with small dog

### PR DESCRIPTION
**Pull request by vieraboschkova**
exchange small dog with big dog

### CODE REVIEW

**Comment by vieraboschkova** [comment]
Clipboard01

File: static/images/avatars/big_dog.svg [hunk 1]
```diff
@@ -1 +1,108 @@
...
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="35"
+   height="35"
+   viewBox="0 0 35 35"
```
Comment (DominikNowak) [inline static/images/avatars/big_dog.svg:10]: Aren't width and height properties included in viewBox?

**Review by DominikNowak — APPROVED** [review]

**Review by DominikNowak — COMMENTED** [review]
I'm not sure if this exchange is neccessary, in my opinion dog with long ears looks for smaller than the second one. I suggest consult this with the PO @Enessetere

**Comment by Enessetere** [comment]
Imo this exchange is unnecessary. I will wait for more votes about this.

**Comment by synowa** [comment]
change: 👍 
do not change: 👎

**Pull request closed without merging**